src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -162,18 +162,27 @@ class ArchiveExplorer extends React.Component {
      */
     sortItemList(itemList) {
         const { sortBy, sortDirection } = this.state;
+        let sortedItems;
 
         if (!sortBy.length) {
             return itemList;
         }
 
-        const sortedItems = itemList.sort((a, b) => {
-            if (typeof a[sortBy] === 'number' && typeof b[sortBy] === 'number') {
-                return a[sortBy] - b[sortBy];
-            }
+        try {
```

Comment:
It seems like we should be able to guard against pretty much any issues without resorting to try/catch here. I think we should add a `componentDidCatch` to the parent component, though.

Reply:
I think we are safe as well, but I through it in for good measure to ensure we always get an itemList returned.

Reply:
Ideally, try/catch should be reserved for exception handling rather than used for flow control. I don't think it hurts in this case, but it can get overused.